spec/slack/web/api/endpoints/custom_specs/auth_spec.rb

```diff
@@ -13,8 +13,13 @@
     end
   end
   context '429 error', vcr: { cassette_name: 'web/429_error' } do
-    it 'fails with an exception' do
-      expect { client.auth_test }.to raise_error Faraday::ClientError
+    it 'fails with an specific exception' do
+      begin
+        client.auth_test
+      rescue Slack::Web::Api::TooManyRequestsError => e
```

Comment:
Note that this new error class does _not_ inherit from `Faraday::ClientError`, so a client is expecting to rescue that error in the case of a 429 response code, this PR will break that behavior.